cli.js

```diff
@@ -13,7 +20,7 @@ Usage:
   parsec-switcher -l | --list
   parsec-switcher -h | --help
   parsec-switcher setup <parsecdLocation>
-  
+  parsec-switcher changeDefault <nickname>
```

Comment:
no need to add this feature, people can simply delete and create new... and if we want to allow renaming, better keep it generic so that they can rename any account... it'll feel more consistent

Reply:
Done